README.md

```diff
@@ -1,134 +1,423 @@
+if both PostgreSQL database and SQLite character data from database file has UTF-8 or UTF-16 encoding. Otherewise
+character interpretation transformation problems will occur.
+
+Character case functions such as `upper`, `lower` and other are not pushed down because they does not work with UNICODE character in SQLite.
+
+Examples
+--------
+
+### Install the extension:
+
+```sql
+	CREATE EXTENSION sqlite_fdw;
+```
+
+### Create a foreign server with appropriate configuration:
+
+Please specify SQLite database path using `database` option.
+
+```sql
+	CREATE SERVER sqlite_server
+	FOREIGN DATA WRAPPER sqlite_fdw
+	OPTIONS (
+          database '/path/to/database'
+	);
+```
+
+### User mapping
+
+There is no user or password conceptions in SQlite, hence `sqlite_fdw` no need any `CREATE USER MAPPING` command.
 
 ### Create foreign table
 Please specify `table` option if SQLite table name is different from foreign table name.
-<pre>
-CREATE FOREIGN TABLE t1(a integer, b text) SERVER sqlite_server OPTIONS (table 't1_sqlite');
-</pre>
+
+```sql
+	CREATE FOREIGN TABLE t1 (
+	  a integer,
+	  b text)
+	SERVER sqlite_server
+	OPTIONS (
+	  table 't1_sqlite'
+	);
+```
 
 If you want to update tables, please add `OPTIONS (key 'true')` to a primary key or unique key like the following:
-<pre>
-CREATE FOREIGN TABLE t1(a integer OPTIONS (key 'true'), b text) SERVER sqlite_server OPTIONS (table 't1_sqlite');
-</pre>
-
-If you need to convert INT SQLite column (epoch Unix Time) to be treated/visualized as TIMESTAMP in PostgreSQL, please add `OPTIONS (column_type 'INT')` when
-defining FOREIGN table at PostgreSQL like the following:
-<pre>
-CREATE FOREIGN TABLE t1(a integer, b text, c timestamp without time zone OPTIONS (column_type 'INT')) SERVER sqlite_server OPTIONS (table 't1_sqlite');
-</pre>
-### Import foreign schema
-<pre>
-IMPORT FOREIGN SCHEMA public FROM SERVER sqlite_server INTO public;
-</pre>
+
+```sql
+	CREATE FOREIGN TABLE t1(
+	  a integer OPTIONS (key 'true'),
+	  b text)
+	SERVER sqlite_server 
+	OPTIONS (
+	  table 't1_sqlite'
+	);
+```
+
+If you need to convert INT SQLite column (epoch Unix Time) to be treated/visualized as `TIMESTAMP` in PostgreSQL, please add `OPTIONS (column_type 'INT')` when defining FOREIGN table at PostgreSQL like the following:
+
+```sql
+	CREATE FOREIGN TABLE t1(
+	  a integer,
+	  b text,
+	  c timestamp without time zone OPTIONS (column_type 'INT'))
+	SERVER sqlite_server
+	OPTIONS (
+	  table 't1_sqlite'
+	);
+```
+
+As above, but with aliased column names:
+
+```sql
+	CREATE FOREIGN TABLE t1(
+	  a integer,
+	  b text OPTIONS (column_name 'test_id'),
+	  c timestamp without time zone OPTIONS (column_type 'INT', column_name 'unixtime')
+	SERVER sqlite_server
+	OPTIONS (
+	  table 't1_sqlite'
+	);
+```
+
+### Import a SQLite database as schema to PostgreSQL:
+
+```sql
+	IMPORT FOREIGN SCHEMA someschema
+	  FROM SERVER sqlite_server;
```

Comment:
- Please fix the indent style here also.  
- The semicolon next to "sqlite_server" is unnecessary.

Reply:
Fixed, https://github.com/mkgrgis/sqlite_fdw/commit/7b7248dbe33e155c101bf11d6af2bfdc36ebf6a3

Reply:
Thank you for fixing.